### PR TITLE
Fixed wire() throwing an exception with a falsy el.

### PR DESCRIPTION
**Pull request by mralex**
- Fixed wire failing when an undefined element is passed in.
- Missing plugins message now triggered from `registeredPlugins()`

### CODE REVIEW

**Comment by kevinastone** [comment]
:ship:

File: dist/dorsal.js [hunk 1]
```diff
@@ -207,7 +207,15 @@ DorsalCore.prototype._runPlugin = function(el, pluginName) {
  * @return {Array} registered plugin names
  */
 DorsalCore.prototype.registeredPlugins = function() {
-    return Object.keys(this.plugins);
+    var pluginKeys = Object.keys(this.plugins);
+
+    if (!pluginKeys.length) {
+        if (console && console.warn) {
```
Comment (gagoar) [inline dist/dorsal.js]: why not using the log system instead of console.warn ?

Comment (gagoar) [inline dist/dorsal.js]: we already use console.log in a controlled way https://github.com/eventbrite/dorsal/blob/master/src/log.js#L48-L59

Comment (mralex) [inline dist/dorsal.js]: I was just done implementing log.warn for this suggestion. Then I noticed there's no way to put Dorsal into debug mode without modifying the constant, effectively making the whole custom logging thing useless until enabling debug mode by Dorsal consumers is implemented.

The message in question is pretty critical to usage of Dorsal, and is useful in a development environment (at least ours).

I'll commit my updates to DorsalLog, enabling `log.info` and `log.warn`, but will keep this explicit `console.warn` until we have ratified enabling debug mode for library consumers. I don't have the cycles at right now to go through that too.

**Comment by gagoar** [comment]
LGTM then :ship:

**Pull request merged**